manifests/install.pp

```diff
@@ -53,13 +57,13 @@
       revision => $version,
     }
   } else {
-    package { 'letsencrypt':
+    package { $package_name:
```

Comment:
Nitpick, I'd suggest to keep the resource title the same (e.g. `Package[letsencrypt]`) but supply the `name` parameter with the varying value, then it's less confusing if `$package_name` isn't set correctly or somebody wants to add a relationship on the resource.